.github/workflows/e2e.yml

```diff
@@ -0,0 +1,123 @@
+#  the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+# Note: Any changes to this workflow would be used only after merging into develop
+name: Build e2e tests
+
+on:
+  push:
+    branches: [ Final-Pr-e2e ]
+  pull_request:
+    branches: [ Final-Pr-e2e ]
+    types: [ opened, synchronize, reopened, labeled ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: k8s-runner-e2e
+    # We allow builds:
+    # 1) When triggered manually
+    # 2) When it's a merge into a branch
+    # 3) For PRs that are labeled as build and
+    #  - It's a code change
+    #  - A build label was just added
+    # A bit complex, but prevents builds when other labels are manipulated
+    if: >
+      github.event_name == 'workflow_dispatch'
+      || github.event_name == 'push'
+      || (contains(github.event.pull_request.labels.*.name, 'build')
+          && (github.event.action != 'labeled' || github.event.label.name == 'build')
+          )
+    strategy:
+      fail-fast: false
+
+    steps:
+      # Pinned 1.0.0 version
+      - uses: actions/checkout@v3
+        with:
+          path: plugin
+          submodules: 'recursive'
+          ref: ${{ github.event.workflow_run.head_sha }}
+
+      - uses: dorny/paths-filter@b2feaf19c27470162a626bd6fa8438ae5b263721
+        if: github.event_name != 'workflow_dispatch' && github.event_name != 'push'
+        id: filter
+        with:
+          working-directory: plugin
+          filters: |
+            e2e-test:
+              - '**/e2e-test/**'
+      - name: Checkout e2e test repo
+        uses: actions/checkout@v3
+        with:
+          repository: cdapio/cdap-e2e-tests
+          path: e2e
+
+      - name: Cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ github.workflow }}-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-${{ github.workflow }}
+      - name: Get Secrets from GCP Secret Manager
+        id: secrets
+        uses: 'google-github-actions/get-secretmanager-secrets@v0'
+        with:
+          secrets: |-
+            ORACLE_HOST:cdapio-github-builds/ORACLE_HOST
+            ORACLE_USERNAME:cdapio-github-builds/ORACLE_NORMAL_USERNAME
+            ORACLE_PASSWORD:cdapio-github-builds/ORACLE_NORMAL_PASSWORD
+            ORACLE_PORT:cdapio-github-builds/ORACLE_PORT
+            PROJECT_ID:cdapio-github-builds/PROJECT_ID_FOR_REPL
```

Comment:
Is this project id different from where the e2e tests are running?

Reply:
Cdf instance run and bq table creation happen on different projects in other e2e tests. So PROJECT_ID points to the other project. Instead of having separate projects,  i thought let's have in single project